test/functions_test.py

```diff
@@ -210,7 +210,7 @@
 >>> loaded_config["output_format"] == input_data_["output_format"]
 True
 >>> loaded_config = check_for_config(input_func_conf_test2)
-Config files detected in current directory are listed below:
+Config files detected in current directory are listed below :
```

Comment:
Also, fix them accordingly.

Reply:
fixed in https://github.com/sepandhaghighi/pyrgg/pull/170/commits/ce9d852ec1025fb4dfaf9fa784824e708712a389